.github/workflows/auto_roll.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ex
+
+browser_name="$1"
+playwright_dir="$(pwd)"
+
+function set_github_action_env {
+    echo "::set-env name=$1::$2"
+}
+
+git config --global user.email "microsoftopensource@users.noreply.github.com"
+git config --global user.name "Microsoft Open Source"
+export SHELL="/bin/bash"
+
+# Ensure the compiled browser will be used
+rm -rf ~/.cache/ms-playwright/
+
+if [ "$browser_name" = "chromium" ]; then
+    ./browser_patches/chromium/build.sh --linux
+    set_github_action_env "CRPATH" "$(pwd)/browser_patches/chromium/output/chrome-linux/chrome"
+    exit 0
+elif [ "$browser_name" = "webkit" ]; then
+    sudo apt install -y libharfbuzz-dev libepoxy-dev libgcrypt-dev libsoup2.4-dev libwebp-dev flatpak
+elif [ "$browser_name" = "firefox" ]; then
+    sudo apt install -y autoconf2.13 libclang-dev clang
+fi
+
+./browser_patches/prepare_checkout.sh "$browser_name"
+
+if [ "$browser_name" = "webkit" ]; then
```

Comment:
nit(here and everywhere): use double-brackets. There's a nice explanation about this in the [google bash styleguide](https://google.github.io/styleguide/shellguide.html#tests) (and it's a nice read altogether).

```sh
if [[ "${browser_name}" == "webkit" ]]; then
fi
```